tasks/parse-pdr/webpack.config.js

```diff
@@ -6,7 +6,18 @@ const root = path.resolve(__dirname);
 const ignoredPackages = [
   'cloudflare:sockets',
   'cpu-features',
-  'sshcrypto.node'
+  'sshcrypto.node',
```

Comment:
We're bringing in the DB package, so this boilerplate was added.

Reply:
Great question - it's coming in from the `ingest` package.   

There's probably room for a question re: if we should split up ingest, or change the import structure.     I'll think on it.